docs/user/discover.asciidoc

```diff
@@ -3,101 +3,210 @@
+. To view the count of documents for a given time in the specified range,
+click and drag the mouse over the histogram, and then click *Refresh*.
 
 [float]
-=== Filter your search results
+[[explore-fields-in-your-data]]
+=== Step 3. Explore the fields in your data
 
-Next, you'll want narrow your search results to a more manageable data set.
-When you click on a name in the field list, you'll see
-the top five values for the field, the number of documents that contain the field,
-and the percentage of documents that contain each value. From this view, you can
-use the (+) magnifier icon to quickly find all
-documents that have that value, or (-) to exclude all
-documents with that value. For more filter options, see <<field-filter, filtering by field>>.
+**Discover** includes a table that shows all the documents that match your search.
+By default, the table includes columns for the time field and the document `_source`,
+which can be overwhelming. You’ll modify this table to display only your fields of interest.
 
+. Scan through the list of **Available fields** to see
+what’s in your data. You can also search for a field by name.
+
+. Find the `manufacturer` field, and then click it to view the five most popular values for that field.
++
 [role="screenshot"]
-image::images/filter-field.png[height=317]
+image:images/filter-field.png[Fields list that displays the top five search results]
+
+. Click image:images/add-icon.png[Add icon] to toggle the field into the document table.
+
+. Add `day of week` so your document table looks like this:
++
+[role="screenshot"]
+image:images/document-table.png[Document table with fields for manufacturer, geo.country_iso_code, and day_of_week]
+
+. To rearrange the table columns, hover the mouse over a
+column header, and then use the move and sort controls.
+
+[float]
+[[search-in-discover]]
+=== Step 4. Search your data
+
+One of the unique capabilities of **Discover** is the ability to combine
+free text search with filtering based on structured data.
+To search all fields, enter a simple string in the **Search** field. To search particular fields and
+build more complex queries, use the <<kuery-query,Kibana Query language>>.
+As you type, KQL prompts you with the fields you can search and the operators
+you can use to build a structured query.
+
+Search the ecommerce data for documents where the country matches US:
+
+. Enter `g`, and then select *geoip.country_iso_code*.
+. Select *equals some value* and *US*, and then click *Update*.
+. For a more complex search, try:
++
+`geoip.country_iso_code : US and products.taxless_price >= 75`
 
+[float]
+[[filter-in-discover]]
+=== Step 5. Filter your data
+
+Whereas the query defines the set of documents you are interested in,
+filters enable you to zero in on different subsets of those documents.
+You can filter results to include or exclude specific fields,
+filter by data type, filter for a value in a range, and more.
```

Comment:
Filter for values, ranges, documents that contain or don't contain fields, yes ... but filtering by data type, I'm not 100% sure what that means in this case